.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - main
+    - stable
+    - v*.*.*
+    tags:
+    - v*
+  pull_request:
+  workflow_dispatch:
+  schedule:
+  - cron: 37 8 * * 1
+
+jobs:
+
+  tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 30
+
+    strategy:
+      fail-fast: false
+
+      matrix:
+        include:
+
+        - name: Tests, Python 3.13, Ubuntu
+          os: ubuntu-latest
+          nox_session: tests-3.13(all)
+          python: '3.13'
+
+        - name: Tests, Python 3.12, with code coverage, macOS
+          os: macos-latest
+          nox_session: tests-3.12(cov)
+          python: '3.12'
+
+        - name: Tests, Python 3.11, lowest direct dependencies, Windows
+          os: windows-latest
+          nox_session: tests-3.11(lowest-direct)
+          python: '3.11'
```

Comment:
Running tests against the lowest versions of direct dependencies will help us ensure that XRTpy works in older environments, which is important for scientific reproducibility.